tests/routes/test_document_families.py

```diff
@@ -728,11 +727,9 @@ def test_update_document__logs_warning_on_four_letter_language(
     assert json_object["languages"] == []
 
     assert (
-        log_spy.call_args_list[0].args[0]
-        == "Retrieved no language from database for meta_data object "
-        "language"
+        "Retrieved no language from database for meta_data object language"
+        in caplog.text
     )
-    assert len(log_spy.call_args_list) == 1
```

Comment:
So this looks like actual code changes not just refactoring, was this introduced by trunq?

Reply:
Indeed introduced by trunk - the error this fixed was specifically:
`715:22  high  Cannot access member "spy" for type "function" ↵   Member "spy" is unknown  pyright/reportFunctionMemberAccess`